clis/teliod/src/core_api.rs

```diff
@@ -0,0 +1,175 @@
+use crate::ClientConfig;
+use base64::{prelude::*, DecodeError};
+use reqwest::{header, Client, StatusCode};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::sync::Arc;
+use telio::crypto::PublicKey;
+use thiserror::Error;
+use tracing::{debug, info};
+
+const API_BASE: &str = "https://api.nordvpn.com/v1";
+
+#[cfg(windows)]
+const OS_NAME: &str = "windows";
+#[cfg(target_os = "macos")]
+const OS_NAME: &str = "macos";
```

Comment:
I don't think there is any point of making this for Windows. Maybe it is ok for MacOS?

Reply:
Removed Windows.